CONTRIBUTORS.md

```diff
@@ -94,6 +94,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
   <tr>
     <td align="center"><a href="https://github.com/JakobFels"><img src="https://avatars.githubusercontent.com/u/4647840?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Jakob Fels</b></sub></a><br /><a href="https://github.com/jbangdev/jbang/issues?q=author%3AJakobFels" title="Bug reports">🐛</a></td>
     <td align="center"><a href="http://davsclaus.com/"><img src="https://avatars.githubusercontent.com/u/477100?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Claus Ibsen</b></sub></a><br /><a href="https://github.com/jbangdev/jbang/issues?q=author%3Adavsclaus" title="Bug reports">🐛</a></td>
+    <td align="center"><a href="https://github.com/basil"><sub><b>Basil Crow</b></sub></a><br /><a href="https://github.com/jbangdev/jbang/commits?author=basil" title="Code">💻</a></td>
```

Comment:
Don't add to this file - that is done by a bot when merge is done.

Reply:
Dropped